tests/Unit/LdapUsersTest.php

```diff
@@ -472,11 +472,6 @@ public function test_getAllUserLogins_ReturnsLdapEntities()
         $this->assertEquals(array(self::TEST_USER, self::TEST_ADMIN_USER), $logins);
     }
 
-    public function test_getExistingLdapUsersFromDb()
```

Comment:
We already have System tests for this scenario.

https://github.com/matomo-org/plugin-LoginLdap/blob/5.x-dev/tests/System/SynchronizeUsersCommandTest.php#L51

https://github.com/matomo-org/plugin-LoginLdap/blob/5.x-dev/tests/System/SynchronizeUsersCommandTest.php#L71